setup.cfg

```diff
@@ -1,5 +1,21 @@
 [flake8]
-exclude = build,.git,build-pypy,.tox
+exclude =
+    .git,
+    __pycache__,
+    docs/source/conf.py,
+    docs/conf.py,
+    old,
+    build,
+    dist,
+    build-pypy,
+    .tox,
+    .eggs,
+    venv
 
 [mypy]
 ignore_missing_imports = True
+
+[pydocstyle]
+
+[pylint]
+disable = R0205
```

Comment:
Please also add pylint to make / GitHub actions, otherwise this will just regress.